### PR TITLE
FileBaseTagsCmd concurrency issue

### PR DESCRIPTION
**Pull request by fpichler**
https://synjira.atlassian.net/browse/CELDEV-271

### CODE REVIEW

File: src/main/java/com/celements/web/plugin/cmd/FileBaseTagsCmd.java [hunk 1]
```diff
@@ -23,42 +23,88 @@
...
   public String getTagSpaceName(XWikiContext context) {
-    String centralFileBaseName = getCentralFileBaseFullName(context) + ".";
-    return centralFileBaseName.substring(0, centralFileBaseName.indexOf('.'));
+    return getTagSpaceRef().getName();
   }
 
+  public SpaceReference getTagSpaceRef() {
+    String centralFileBaseName = getCentralFileBaseFullName() + ".";
+    String spaceName = centralFileBaseName.substring(0, centralFileBaseName.indexOf('.'));
+    return new SpaceReference(spaceName, new WikiReference(getContext().getDatabase()));
```
Comment (msladek) [inline src/main/java/com/celements/web/plugin/cmd/FileBaseTagsCmd.java]: this statements throws an IAE if getCentralFileBaseFullName() returns the default empty string

**Comment by msladek** [comment]
please add test for getTagSpaceRef() where getSpacePreference() returns empty string and fix it.

looks good otherwise :)

**Comment by fpichler** [comment]
issue fixed. Thanks for the hint.
